build.gradle

```diff
@@ -30,5 +30,5 @@ task clean(type: Delete) {
 ext {
     supportLibraryVersion = '27.1.1'
     butterknifeVersion = '8.8.1'
-    daggerVersion = '2.16'
-}
\ No newline at end of file
+    daggerVersion = '2.19'
+}
```

Comment:
Still, the same issue, remove this line change.

Reply:
that warning "no newline at end of file" is coming by default

Reply:
I think this is fine for now

Reply:
ok